### PR TITLE
Comments from the Code Engine DUX review

### PR DESCRIPTION
**Pull request by kersten1**
Hello,
We received the following comments from the latest Code Engine DUX review:

- [x] Cloud shell is available in only 3 regions.
- [ ] Project and cloud shell location (region) must be the same 
- [x] When you create a bucket, add the date to your initials to insure uniqueness
- [x] Mention that you can also complete many of these tasks with the UI (sprinkled throughout topic)
- [x] Add step for checkbox when you create the NLU

Please contact me if you want to discuss any of these updates. 

Thank you!

### CODE REVIEW

**Comment by l2fprod** [comment]
@kersten1 

Comments about Cloud Shell are incorrect.

Although your Cloud Shell instance (the shell window) is running in a specific region, once you are in cloud shell you can use `ibmcloud target -r <region>` to target any region API so the review comments about Cloud Shell are invalid.

**Comment by kersten1** [comment]
Good to know! Let me confirm back with my people and I will update PR.

**Comment by l2fprod** [comment]
@kersten1 

from https://cloud.ibm.com/docs/cloud-shell?topic=cloud-shell-issues-limitations

image

File: text-analysis-code-engine.md [hunk 1]
```diff
@@ -69,27 +69,27 @@ This tutorial requires:
...
 
-Putting entities into a single project enables you to manage access control more easily. The entities within a project share the same private network, which enables them to talk to each other securely. To understand what a project is, check the [documentation](https://{DomainName}/docs/codeengine?topic=codeengine-manage-project)
+Putting entities into a single project enables you to manage access control more easily. The entities within a project share the same private network, which enables them to talk to each other securely. To understand what a project is, check the [documentation](https://{DomainName}/docs/codeengine?topic=codeengine-manage-project). 
 
 1. Navigate to [{{site.data.keyword.codeenginefull_notm}} Overview](https://{DomainName}/codeengine/overview) page.
 2. On the left pane, click on **Projects** and then click **Create**.
-   - Select a location.
+   - Select a location. You must use the same region location for both your project and your {{site.data.keyword.cloud-shell_short}} instance. For more information, see [{{site.data.keyword.cloud-shell_short}}](/docs/cloud-shell?topic=cloud-shell-shell-ui#change-location) documentation.
```
Comment (kersten1) [inline text-analysis-code-engine.md:92]:
```suggestion
   - Select a location. 
```

**Comment by kersten1** [comment]
@l2fprod Thank you for that info! I updated the PR to remove that information.

**Comment by l2fprod** [comment]
capture feedback in our private repo

**Pull request closed without merging**